packages/plugins/i18n/server/services/metrics.js

```diff
@@ -11,13 +11,13 @@ const sendDidInitializeEvent = async () => {
     0
```

Comment:
those actions are made by users

Reply:
They also fire every time the server starts, hence they are anonymous. Is that the desired behavior do you know?